tests/mysql/query.rs

```diff
@@ -1161,7 +1183,7 @@ fn insert_on_conflict_0() {
         [
             r#"INSERT INTO `glyph` (`aspect`, `image`)"#,
             r#"VALUES ('04108048005887010020060000204E0180400400', 3.1415)"#,
-            r#"ON DUPLICATE KEY UPDATE `aspect` = VALUES(`aspect`), `image` = VALUES(`image`)"#,
+            r#"ON DUPLICATE KEY UPDATE `aspect` = `aspect`, `image` = `image`"#,
```

Comment:
Oh sad, this is wrong. `ON DUPLICATE KEY UPDATE id = id` should only applies to `do_nothing_on`.
While the behaviour of `update_columns()` should not be changed.
The two expressions have **different semantics**.